components/wb-fieldflow/fieldflow.js

```diff
@@ -1207,9 +1207,14 @@ $document.on( "submit", selectorForm + " form", function( event ) {
 						cacheName = items[ 0 ];
 						cacheParam = items[ 1 ];
 					}
-					$hdnField = $( "<input type='hidden' name='" + cacheName + "' value='" + wb.escapeAttribute( cacheParam ) + "' />" );
-					$elm.append( $hdnField );
-					wbRegisteredHidden.push( $hdnField.get( 0 ) );
+
+					hdnField = document.createElement( "input" );
+					hdnField.setAttribute( "type", "hidden" );
+					hdnField.setAttribute( "name", cacheName );
+					hdnField.setAttribute( "value", wb.escapeAttribute( cacheParam ) );
```

Comment:
Should I get rid of the call to [``wb.escapeAttribute()``](https://github.com/wet-boew/wet-boew/blob/e01bc308c1cc123dbe7948cd8dccc14829ed8e70/src/core/helpers.js#L1313-L1315) at this spot?

I don't think ``setAttribute()`` technically has a need for it, but figured it can't hurt to ask before going through with it.

Reply:
Sorry - must've overlooked this :(.

In any case, I ended up implementing @duboisp's suggestion to set the attribute properties directly instead of using ``setAttribute()`` later on. Given that, I think it might be a safer bet to keep ``wb.escapeAttribute()`` in play.

I won't bother sending another PR to remove it.